test/blackbox-tests/test-cases/findlib-error/in-jbuild/jbuild

```diff
@@ -0,0 +1,5 @@
+(jbuild_version 1)
+
+(rule
+(write-file target.txt ${findlib:pkg:file})
```

Comment:
Btw, it would be nice to standardize on the formatting here. Have a look at the other dune files for how format this stuff.